R/assign_job_queue.R

```diff
@@ -113,53 +128,58 @@ get_proc_medians <- function(dir_job_results) {
 #'
 #' @return [tbl_df] 2 columns: 1) process and 2) median seconds
 #'
-#' example: write_proc_medians_table(
+#' example: writeProcessRuntime2TSV(
 #'   "/data/scratch/janani/molevolvr_out/",
 #'   "/data/scratch/janani/molevolvr_out/log_tbl.tsv"
 #' )
 #' @export
-write_proc_medians_table <- function(dir_job_results, filepath) {
-    df_proc_medians <- get_proc_medians(dir_job_results) |>
-        tibble::as_tibble() |>
-        tidyr::pivot_longer(
-            dplyr::everything(),
-            names_to = "process",
-            values_to = "median_seconds"
-        ) |>
-        dplyr::arrange(dplyr::desc(median_seconds))
-    readr::write_tsv(df_proc_medians, file = filepath)
-    return(df_proc_medians)
+writeProcessRuntime2TSV <- function(dir_job_results, filepath) {
+  df_proc_medians <- calculateProcessRuntime(dir_job_results) |>
+    tibble::as_tibble() |>
+    tidyr::pivot_longer(
+      dplyr::everything(),
+      names_to = "process",
+      values_to = "median_seconds"
+    ) |>
+    dplyr::arrange(dplyr::desc(median_seconds))
+
+  # Write the resulting tibble to a TSV file
+  readr::write_tsv(df_proc_medians, file = filepath)
+  return(df_proc_medians)
 }
 
+#' writeProcessRuntime2YML
+#' 
+#' @description
 #' Compute median process runtimes, then write a YAML list of the processes and
 #' their median runtimes in seconds to the path specified by 'filepath'.
 #'
 #' The default value of filepath is the value of the env var
-#' MOLEVOLVR_PROC_WEIGHTS, which get_proc_weights() also uses as its default
+#' MOLEVOLVR_PROC_WEIGHTS, which writeProcessRuntime2YML() also uses as its default
 #' read location.
 #'
 #' @param dir_job_results [chr] path to MolEvolvR job_results directory
-#' @param filepath [chr] path to save YAML file; if NULL, uses ./molevol_scripts/log_data/job_proc_weights.yml
+#' @param filepath [chr] path to save YAML file; if NULL, 
+#'                 uses ./molevol_scripts/log_data/job_proc_weights.yml
 #'
 #' @importFrom yaml write_yaml
 #'
 #' @examples
 #' \dontrun{
-#' write_proc_medians_yml(
+#' writeProcessRuntime2YML(
 #'     "/data/scratch/janani/molevolvr_out/",
 #'     "/data/scratch/janani/molevolvr_out/log_tbl.yml"
 #' )
 #' }
 #' @export
-write_proc_medians_yml <- function(dir_job_results, filepath = NULL) {
-    if (is.null(filepath)) {
-        filepath <- file.path(common_root, "molevol_scripts", "log_data", "job_proc_weights.yml")
-    }
```

Comment:
This check appears to be important; why was it removed?

Reply:
Resolved with [df602df](https://github.com/JRaviLab/MolEvolvR/pull/95/commits/df602dfd63cbab0d84dbcc8229e3da9c7646b9d5)